src/pudl/output/eia923.py

```diff
@@ -70,7 +70,11 @@ def generation_fuel_eia923(
     if end_date is not None:
         gf_select = gf_select.where(gf_tbl.c.report_date <= end_date)
 
-    gf_df = pd.read_sql(gf_select, pudl_engine)
+    gf_df = (
+        pd.read_sql(gf_select, pudl_engine)
+        .rename(columns={"eia_sector": "sector_id_eia"})
+        .rename(columns={"sector_name": "sector_name_eia"})
+    )
```

Comment:
What led you to renaming these columns in output functions for `generation_fuel_eia923`, `boiler_fuel_eia923`, and `generation_eia923`?

Reply:
@knordback Should these still be getting renamed with the new column names? Do those columns exist? (Also in the `boiler_fuel_eia923` and `generation_eia923` tables below)